pyproject.toml

```diff
@@ -52,7 +52,7 @@ dependencies = [
 dynamic = ["version"]
 
 [project.readme]
-file = "LONG_DESCRIPTION.rst"
+file = "README.rst"
 content-type = "text/x-rst"
```

Comment:
Do we really need to specify `content-type`?

Reply:
I think it's best practice, but I am not certain.